client/src/components/History/Content/model/states.js

```diff
@@ -19,12 +19,17 @@ export const STATES = {
         status: "success",
         text: "No data.",
     },
-    /** the tool producing this dataset failed */
+    /** the tool producing this dataset has errored */
     error: {
         status: "danger",
         text: "An error occurred with this dataset:",
         icon: "exclamation-triangle",
     },
+    /** the dataset has failed */
+    failed: {
+        status: "danger",
+        icon: "exclamation-triangle",
+    },
```

Comment:
It's a valid job state, yes

Reply:
But that means that it should be mapped and should remain in the list.

Reply:
Well now you're saying it's a dataset state, which is not correct.

Reply:
Its not about it being a dataset state, its that it is in `job_state_summary` and if encountered returns `{ status: "danger", icon: "exclamation-triangle" }` so that the reduced collection state can be displayed correctly.

Reply:
It's a job state, and here you are listing dataset states.  a new developer coming in will not possibly understand what the hell is going on here if you mix states like that.